script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -61,22 +71,24 @@ Below them is the crater that had become a battlefield
 [sha:e67efe6edeb4c986f11fe524af54f53930a56854]{
 -- Page 789, Offset 12997.
 -- 見れば、クレーターではいくつかの人影がせわしなく活動していた。
-If they looked down, they would see several figures hurrying about their work inside the crater.
+If they were to look down, they would see a handful of figures beavering away inside it.
 }
 [sha:9bbd9f9f454501d5fe6da28d9647b693c9f72085]{
 -- Page 789, Offset 12998.
 -- 災害現場の人命救助に出動したレスキュー隊、および所轄の警察から派遣された調査団だ。
-There is the rescue unit dispatched to save lives at the disaster site, as well as the investigation team dispatched from the relevant police section.
+Some are a members of a search and rescue unit dispatched to the disaster site,
```

Comment:
remove 'a'

Reply:
unless "units" becomes plural, this suggestion doesn't work. In the CG is there a lot of them?